docs/usage/shoot_pod_autoscaling_best_practices.md

```diff
@@ -0,0 +1,337 @@
+
+VPA doesn't offer many more settings that can be tuned per VPA resource than you see above (different than HPA's `behavior` section). However, there is one more that isn't shown above, which allows to [scale only up or only down (`evictionRequirements[].changeRequirement`)](https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/README.md#controlling-eviction-behavior-based-on-scaling-direction-and-resource), in case you need that, e.g. to provide resources when needed, but avoid disruptions otherwise.
+
+## VPA Options
+
+VPA is an independent community project that [consists of](https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/README.md#components-of-vpa) a recommender (computing target recommendations and bounds), an updater (evicting pods that are out of recommendation bounds), and an admission controller (mutating webhook applying the target recommendation to newly created pods). As such, they have independent options.
+
+### VPA Recommender Options
+
+You can read up the [full VPA recommender options](https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/FAQ.md#what-are-the-parameters-to-vpa-recommender) online and set some of them conveniently in your [Gardener shoot cluster spec](https://github.com/gardener/gardener/blob/957e4c7/example/90-shoot.yaml#L298-L307):
+- `recommendationMarginFraction` (default 15%): Safety margin that will be added to the recommended requests.
+- `targetCPUPercentile` (default 90%): CPU usage percentile that will be targeted with the CPU recommendation (i.e. recommendation will "fit" e.g. 90% of the observed CPU usages). This setting is relevant for balancing your requests reservations vs. your costs. If you want to reduce costs, you can reduce this value (higher risk because of potential under-reservation, but lower costs), because CPU is compressible, but then VPA may lack the necessary signals for scale-up as throttling on an otherwise fully utilized node will go unnoticed by VPA. If you want to err on the safe side, you can increase this value, but you will then target more and more a worst case scenario, quickly (maybe even exponentially) increasing the costs.
+- `targetMemoryPercentile` (default 90%): Memory usage percentile that will be targeted with the memory recommendation (i.e. recommendation will "fit" e.g. 90% of the observed memory usages). This setting is relevant for balancing your requests reservations vs. your costs. If you want to reduce costs, you can reduce this value (higher risk because of potential under-reservation, but lower costs), because OOMs will trigger bump-ups, but those will disrupt the workload. If you want to err on the safe side, you can increase this value, but you will then target more and more a worst case scenario, quickly (maybe even exponentially) increasing the costs.
```

Comment:
we don't expose `targetMemoryPercentile` at the moment, should we still mention this here?

Reply:
I am very much behind with my personal backlog after the workshops and picking up the threads, but if you remember, I have asked you about this (and recommendationMarginFraction) and I am very interested in this feature. I already mentioned both here, because I am of the opinion, we should expose both and use them for our own landscapes as well.